test/services/AppealServiceSpec.scala

```diff
@@ -42,18 +41,19 @@ import java.time.{LocalDate, LocalDateTime}
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.Random
 
-class AppealServiceSpec extends SpecBase with LogCapturing {
+class AppealServiceSpec extends SpecBase with LogCapturing with FeatureSwitching {
```

Comment:
Looks like we're missing a RegimeAppealsService. Imagine it will be essentially identical to this one till we delete the non-regime version post-migration